spec/support/helper_methods.rb

```diff
@@ -113,6 +113,39 @@ def place_and_complete_item_order(ordered_by, facility, account = nil, reviewed
 end
 
 #
+# Simulates placing an order for a service and having it marked complete
+# [_ordered_by_]
+#   The user who is ordering the service
+# [_facility_]
+#   The facility with which the order is placed
+# [_account_]
+#   The account under which the order is placed
+# [_reviewed_]
+#   true if the completed order should also be marked as reviewed, false by default
+def place_and_complete_service_order(ordered_by, facility, account = nil, reviewed = false)
```

Comment:
Is what's happening here possible with the `:purchased_order` factory? If you can leverage that, or extend it with a `trait`, I'd take that over a helper method.